frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-community-provider-modal.tsx

```diff
@@ -9,81 +9,73 @@ import {
   ModalSubmitFooter,
 } from '@console/internal/components/factory/modal';
 import { ExternalLink } from '@console/internal/components/utils';
+import { useTranslation } from 'react-i18next';
 
-export class OperatorHubCommunityProviderModal extends React.Component<
-  OperatorHubCommunityProviderModalProps,
-  OperatorHubCommunityProviderModalState
-> {
-  constructor(props) {
-    super(props);
-    this.state = {
-      ignoreWarnings: false,
-    };
-  }
+export const OperatorHubCommunityProviderModal: React.FC<OperatorHubCommunityProviderModalProps> = ({
+  close,
+  showCommunityOperators,
+}) => {
+  const { t } = useTranslation();
+  const [ignoreWarnings, setIgnoreWarnings] = React.useState(false);
 
-  onIgnoreChange = (checked) => {
-    this.setState({ ignoreWarnings: checked });
-  };
+  const onIgnoreChange = React.useCallback((checked) => {
```

Comment:
Is there a reason we can't use `setIgnoreWarnings` directly? `onIgnoreChange` seems to wrap that with no other changes?

Reply:
Probably just left behind from the class component implementation. I'll fix this.